ui/00_message/skill/custom_skill_command_08.xml

```diff
@@ -86,9 +86,13 @@ and then unleash with &lt;KC SHOOT&gt;</translation>
   <message>
     <key>CUSTOM_SKILL_COMMAND_08_184</key>
     <original>セットボタンを押して詠唱後、&lt;KC SHOOT&gt;で放つ</original>
+    <translation>Press the set button to begin casting,
```

Comment:
Why the change from "assigned" to "set"? (I won't repeat this message for the others)

Reply:
I just copy paste the same line from previous command in the same file. Should it be assigned accross all classes?

Reply:
I'll say yes, simply because "assigned" sounds much better than "set".

Reply:
Alright I'll change them.